style.css

```diff
@@ -524,3 +564,289 @@ footer ul {
 footer img {
   width: 1.5rem;
 }
+
+@media (min-width: 768px) {
+  body {
+    overflow-x: visible;
+  }
+
+  .main-navbar-menu img {
+    visibility: hidden;
+  }
+
+  .righthand-of-logo {      /* Come back to the listing style and the align item */
+    visibility: visible;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    list-style: none;
+    gap: 30px;
+    height: 100%;
+
+    /* border: 2px solid green; */
+  }
+
+  .logo-right-whole {
+    /* border: 2px solid pink; */
+    text-decoration: none;
+    display: flex;
+    text-align: center;
+  }
+
```

Comment:
- [ ] Please remove all commented (unused) code inside your CSS file to make your code `DRY` and readable.